### DISCUSSION TITLE
Make Validator Requirements Less Than 500 Avax

### DISCUSSION THREAD

**Discussion by CaptainDoubleBlade** [avalanche-foundation/ACPs#56] [Brainstorming]
Bruh we need to do something to make it so regular people are able to join in the decentralization of the Avax chain.  Right meow it costs $88,000 USDC to become a validator and that is a significant amount of capital to keep our chain decentralized.  While it is true we are at an all time high for the amount of validators there are on the chain I believe we can do even better if the requirements were less cost intensive.  I believe 500 Avax or less would be appropriate and I'm open to a discussion on what the right amount could be.  Personally I think 500 should suffice as it is a serious amount but a compromise for those who would say it's too cheap and it doesn't show a serious commitment to keeping the chain decentralized.  Even this amount is worrying because if Avax were to be back at all time high regular people would be priced out again but it is a step in the right direction and I will definitely be here again trying to lower it once again. 
Nonetheless, 500 Avax will be 1/4 the cost of starting a validator and that's meowable.

**Comment by gatesyp** [comment]
meow!
